src/database/seeds/ScrapedDatasetSeeder.php

```diff
@@ -11,11 +11,11 @@ class ScrapedDatasetSeeder extends Seeder
      */
     public function run()
     {
-        factory(ScrapedDataset::class, 2)->create();
+        ScrapedDataset::factory(2)->create();
```

Comment:
Could you please be consistent and use `factory()->count(2)` instead of `factory(2)` ? Check other places just in case.

Reply:
This is done

Reply:
Done